src/components/models-slm.html

```diff
@@ -0,0 +1,63 @@
+<section class="models-slm" data-scroll-trigger="anim-group">
+	<div class="l-container models-slm__container js-anim-group">
+		<div class="divider-label models-slm__label js-anim-item">
+			<div class="divider-label__ill" aria-hidden="true">
+				{{> illustrations/divider-sm-left}}
+			</div>
+			<p class="divider-label__text h5">Models</p>
+			<div class="divider-label__ill" aria-hidden="true">
+				{{> illustrations/divider-sm-right}}
+			</div>
+		</div>
+
+		<p class="h2 models-slm__title js-anim-item">Prem's Specialized SLMs</p>
+		<p class="models-slm__text js-anim-item">
+			A Suite of Foundational Open Source SLMs to Supercharge Business
```

Comment:
@filopedraz  can we review if the copy here, I mean how fonts are capitalised, looks good to me, but from a copy standpoint.
![Screenshot 2024-06-03 at 6 56 44 PM](https://github.com/premAI-io/prem-website/assets/23050213/d923077f-26b9-4d6e-8ef2-d2da4e4b2807)